ts/webui/src/components/experiment/trialdetail/chart/DefaultMetricPoint.tsx

```diff
@@ -118,8 +122,32 @@ class DefaultPoint extends React.Component<DefaultPointProps, DefaultPointState>
         };
     }
 
+    private formatAccuracy(accuracy: number | undefined): number {
+        if (accuracy === undefined || isNaN(accuracy) || Object.is(accuracy, Infinity)) {
```

Comment:
`Object.is(accuracy, Infinity)` -> `!isFinite(accuracy)`
`accuracy` might be `-Infinity`.

Reply:
ok!